src/app/(BottomTabNavigation)/Profile/LogOut/styles.ts

```diff
@@ -0,0 +1,76 @@
+import { StyleSheet } from 'react-native';
+
+import { colors } from '../../../../styles/colors';
+
+export default StyleSheet.create({
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    backgroundColor: colors.white,
+    width: '100%',
+    height: '100%',
+    justifyContent: 'flex-end',
+  },
+  screenContainer: {
+    paddingTop: 32,
+    paddingBottom: 32,
+    paddingLeft: 29,
+    paddingRight: 29,
```

Comment:
Can use `paddingVertical` and `paddingHorizontal` if the dimensions are the same on each side. You can do the same thing for margins as well.